src/Tool/Tool.php

```diff
@@ -84,7 +84,7 @@ public function isEmpty(): bool
      *
      * @psalm-return Generator<ToolVersionInterface>
```

Comment:
Should we change this to `@return Traversable<ToolVersionInterface>` as IDEs have now catched up with the generic notations?